src/main/java/net/pms/database/MediaTableFailedLookups.java

```diff
@@ -137,39 +147,44 @@ private static void createTable(final Connection connection) throws SQLException
 	 */
 	public static boolean hasLookupFailedRecently(final Connection connection, final String fullPathToFile, final boolean isVideo) {
 		boolean removeAfter = false;
-		String latestVersion;
-		if (isVideo) {
-			latestVersion = APIUtils.getApiDataVideoVersion();
-		} else {
-			latestVersion = APIUtils.getApiDataSeriesVersion();
+		String latestVersion = null;
+		if (CONFIGURATION.getExternalNetwork()) {
```

Comment:
Those `APIUtils` methods already handle `getExternalNetwork` so I'm not sure why this is being done here, can you explain it?

Reply:
line 148 was : `if (latestVersion != null && CONFIGURATION.getExternalNetwork()) {`
just changed logic order (test before fill latest version)